py/pysparkling/__init__.py

```diff
@@ -5,7 +5,18 @@
 pySparkling - The Sparkling-Water Python Package
 =====================
 """
-__version__ = "SUBST_PROJECT_VERSION"
+import zipfile
+from codecs import open
+from os import path
+
+here = path.dirname(path.abspath(path.dirname(__file__)))
+if '.zip' in here:
```

Comment:
Did you test this path?

Reply:
I test the zip path but right now I discovered few additional issues which I need to fix. 

Will let you know when this PR is completely ready

Reply:
Tested both paths, also tested via testpypi to ensure that pip works